### ISSUE TITLE
Skip attachements section when loading GeoJSON generated by NextGIS Mobile

### ISSUE THREAD

**Issue by RajeshkumarGA** [nextgis/nextgisweb#560]
When uploading a geojson file created by NEXTGIS Mobile app in NEXTGIS web, i am getting this error.

> Error message: Unknown exception "KeyError" in serializer "vector_layer" attribute "source".
Spatial Reference System: WGS 84/Pseudo-Mercator (EPSG:3857)
Encoding: UTF-8
geojson file contents:
`{
  "crs": {
    "type": "name",
    "properties": {
      "name": "urn:ogc:def:crs:EPSG::3857"
    }
  },
  "type": "FeatureCollection",
  "features": [{
    "type": "Feature",
    "properties": {
      "FID": 123456,
      "TEXT": "test",
      "attaches": []
    },
    "geometry": {
      "type": "Polygon",
      "coordinates": [
        [
          [9708563, 2710463.25],
          [9702726, 2701523.75],
          [9715415, 2703592.25],
          [9702361, 2696059.25],
          [9716653, 2693465],
          [9723996, 2709837.25],
          [9718761, 2721735.25],
          [9705351, 2717464.5],
          [9708563, 2710463.25]
        ]
      ]
    }
  }]
}
`

**Comment by simgislab** [comment]
confirmed, GeoJSON above works well in QGIS

**Comment by drnextgis** [comment]
Confirmed, but it is not looks like a bug, sort of limitation current NextGIS Web data model. As you can see your data contains `attaches` property with value of list type. Now NGW doesnt support such type of properties because it is not clear how we should to store them. @RajeshkumarGA you can delete this property, I thik it should solve youre problem.

**Comment by hridaydutta123** [comment]
@drnextgis Removed the attaches property but its still showing the following error: 

```
Error message: Unknown exception 'ProgrammingError' in serializer 'vector_layer' attribute 'source'.
```

**Comment by drnextgis** [comment]
Cannot confirm, all eorks fine here, [example](http://rykovd.nextgis.com/resource/273). GeoJSON data:

```js
{ "crs": { "type": "name", "properties": { "name": "urn:ogc:def:crs:EPSG::3857" } }, "type": "FeatureCollection", "features": [{ "type": "Feature", "properties": { "FID": 123456, "TEXT": "test"}, "geometry": { "type": "Polygon", "coordinates": [ [ [9708563, 2710463.25], [9702726, 2701523.75], [9715415, 2703592.25], [9702361, 2696059.25], [9716653, 2693465], [9723996, 2709837.25], [9718761, 2721735.25], [9705351, 2717464.5], [9708563, 2710463.25] ] ] } }] }
```

**Comment by hridaydutta123** [comment]
@drnextgis Thanks for the reply. I think i am having problem with installation of the correct version of gdal. Can you mention the version of gdal you are using?

**Comment by drnextgis** [comment]
My link above to instance with GDAL 1.10. But it should also work with another GDAL versions.

**Comment by hridaydutta123** [comment]
@drnextgis Well i'm building nextgis web for Raspberry Pi-3 with Raspian OS. Is there any kind of compatibility issues with this platform?

**Comment by simgislab** [comment]
we have no idea

**Comment by hridaydutta123** [comment]
@simgislab ok.. We are trying to see if there are any dependency issue. We will keep you posted.

**Comment by hridaydutta123** [comment]
When uploading your working GEOJSON data i.e. ```{ "crs": { "type": "name", "properties": { "name": "urn:ogc:def:crs:EPSG::3857" } }, "type": "FeatureCollection", "features": [{ "type": "Feature", "properties": { "FID": 123456, "TEXT": "test"}, "geometry": { "type": "Polygon", "coordinates": [ [ [9708563, 2710463.25], [9702726, 2701523.75], [9715415, 2703592.25], [9702361, 2696059.25], [9716653, 2693465], [9723996, 2709837.25], [9718761, 2721735.25], [9705351, 2717464.5], [9708563, 2710463.25] ] ] } }] }``` 

Now it is showing the error: 

`ProgrammingError: (ProgrammingError) type "geometry" does not exist
LINE 4:  geom geometry(POLYGON,3857), 
`

**Comment by drnextgis** [comment]
Did you install PostGIS into your database?

**Comment by hridaydutta123** [comment]
Ya.. `psql -h localhost -d db_ngw -U ngw_admin -c "SELECT PostGIS_Full_Version();"` returns ` POSTGIS="2.1.4 r12966" GEOS="3.4.2-CAPI-1.8.2 r3921" PROJ="Rel. 4.8.0, 6 March 2012" GDAL="GDAL 1.10.1, released 2013/08/26" LIBXML="2.9.1" LIBJSON="UNKNOWN" RASTER
`

**Comment by drnextgis** [comment]
Fixed via https://github.com/nextgis/nextgisweb/commit/1485fabdc80923ea58630a71b34aca54ac748fb6
